### PR TITLE
css-brochure-form - Sina Daryaei

### PR DESCRIPTION
**Pull request by sinadaryaei**
image

### CODE REVIEW

File: css-brochure-form/index.html [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div>
+    <div>
+      <h3>learn more</h3>
+    </div>
+    <div>
+      <p>company email</p>
+      <input type="text" name="Name" placeholder="sally@company.co" />
```
Comment (shawnkost) [inline css-brochure-form/index.html:19]: This `input` needs a `label` associated with it. I would turn the text on line 18 into a `label`, and link the two using a `for` attribute.

File: css-brochure-form/index.html [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    <div>
+      <p>name</p>
+      <input type="text" name="Name" placeholder="Sally Smith"/>
```
Comment (shawnkost) [inline css-brochure-form/index.html:23]: This `input` needs a `label` associated with it. I would turn the text on line 22 into a `label`, and link the two using a `for` attribute.

File: css-brochure-form/index.html [hunk 3]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    <div>
+      <p>where are you hiring&#63;</p>
+      <input type="text" name="Name" placeholder="City, Country" />
```
Comment (shawnkost) [inline css-brochure-form/index.html:27]: This `input` needs a `label` associated with it. I would turn the text on line 26 into a `label`, and link the two using a `for` attribute.

File: css-brochure-form/index.html [hunk 4]
```diff
@@ -0,0 +1,40 @@
...
+    </div>
+    <div>
+      <p>how many engineers are you hiriing in the next 6 months&#63;</p>
+      <select name="Select an option">
```
Comment (shawnkost) [inline css-brochure-form/index.html:31]: This `select` needs a `label` associated with it. I would turn the text on line 30 into a `label`, and link the two using a `for` attribute.

**Review by shawnkost — CHANGES_REQUESTED** [review]
Some changes requested here Sina!

**Comment by sinadaryaei** [comment]
Thank you!

**Review by shawnkost — APPROVED** [review]
👍🏻

**Pull request merged**